### PR TITLE
[doc] local running

### PR DESCRIPTION
**Pull request by zchcai**
The documentation update for users to run local experiments on their own machines.

Related to #251.

This will open for review once there will be no configuration option change.

Currently, it will be blocked until #445 resolved.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Heh, not sure if this should land before the queue stuff which will set local experiments a few weeks back
:)

**Comment by zchcai** [comment]
> Heh, not sure if this should land before the queue stuff which will set local experiments a few weeks back
> :)

Maybe we can try to let both working together. :)

**Comment by jonathanmetzman** [comment]
> > Heh, not sure if this should land before the queue stuff which will set local experiments a few weeks back
> > :)
> 
> Maybe we can try to let both working together. :)

Heh, wouldn't count on it.

**Comment by zchcai** [comment]
After #461 merged, this can be open for merging.

File: docs/running-a-local-experiment/running_a_local_experiment.md [hunk 1]
```diff
@@ -51,9 +51,11 @@ max_total_time: 86400
 # docker_registry: lab-server:5000
 
 # The local experiment folder that will store most of the experiment data.
+# Please use absolute path.
```
Comment (jonathanmetzman) [inline docs/running-a-local-experiment/running_a_local_experiment.md:54]: nit: "Please use an absolute path".

**Comment by zchcai** [comment]
Discussion offline: though we already add `docker_registry` option for local running (#461), we will not publish our local support documentation until we have at least a basic resource control option for local running.
Blocking until further enhancement. Probably after `queue` branch merged, `queue` branch works on local, and, works for builders and runners together with measures.
For other internal users, they can checkout [this commit](https://github.com/google/fuzzbench/commit/b16c35cb5e819c8d025a6857e7266b457ae5b889) and refer to here for a local run.

**Comment by inferno-chromium** [comment]
@jonathanmetzman - can you take a final review pass and commit this.

**Comment by inferno-chromium** [comment]
actually i will review it and add that line on resource control.

**Pull request merged**